python/lsst/rubintv/production/redisUtils.py

```diff
@@ -513,6 +516,71 @@ def pushNewExposureToHeadNode(self, expRecord):
         expRecordJson = expRecord.to_simple().json()
         self.redis.lpush(queueName, expRecordJson)
 
+    def announceResultInConsDb(self, instrument, table, obsId):
+        """
+        Parameters
+        ----------
+        instrument : `str`
+            The instrument name.
+        table : `str`
+            The table name.
+        obsId : `int`
+            The obsId that was used in the consDbClient.insert() call.
+        """
+        # The call to .lower() is because consDB if all lower case, and we
+        # don't want to be sensitive to table capitization or regular butler
+        # instrument name capitilisation.
+        key = f"consdb-{instrument}-{table}-{obsId}".lower()
+        self.redis.lpush(key, 1)
```

Comment:
I would use `lset()` to ensure that the list never gets to be longer than 1 element.

Reply:
I tried that briefly and found it awkward, and I can't remember why now. I'll try this again before merging (though given that it expires and there's not _that_ many images each night I don't think it matters a _huge_ amount).

Reply:
FWIW, the issue was that `lset()` doesn't create a key that doesn't key exist (unlike `lpush()`) so to stop this growing you need to do
```
-        self.redis.lpush(key, 1)
+
+        if not self.redis.exists(key):
+            self.redis.lpush(key, 1)
```
(or the same with `lset()` but that's just more verbose and results in the same)